tests/unit/models/test_model_interfaces.py

```diff
@@ -452,6 +452,15 @@ def test_sparse_variational_model_attribute() -> None:
     assert sv.model is model
 
 
+def test_sparse_variational_update_updates_num_data() -> None:
+    model = SparseVariational(
+        _svgp(tf.zeros([1, 4])),
+        Dataset(tf.zeros([3, 4]), tf.zeros([3, 1])),
+    )
+    model.update(Dataset(tf.zeros([5, 4]), tf.zeros([5, 1])))
+    assert model.model.num_data == 5
+
+
 @pytest.mark.parametrize(
```

Comment:
This is good for now. The SVGP stuff will be revamped in the next couple of weeks anyway :)